projects/hipsparse/clients/common/CMakeLists.txt

```
@@ -0,0 +1,95 @@
+# Copyright Advanced Micro Devices, Inc., or its affiliates.
+# SPDX-License-Identifier: MIT
+
+# Create common helper target for all clients
+add_library(hipsparse-clients-common OBJECT)
```

```suggestion
add_library(hipsparse-clients-common OBJECT)
add_library(hipsparse::clients-common ALIAS hipsparse-clients-common)
```

Also not sure if we want OBJECT library here. OBJECT libraries have different rules for propagating requirements.

---

Use STATIC instead.